tests/integration/relations/test_opensearch_provider.py

```diff
@@ -452,72 +462,8 @@ async def test_admin_relation(ops_test: OpsTest):
     assert set(artists) == {"Herbie Hancock", "Lydian Collective", "Vulfpeck"}
 
 
+@pytest.mark.group(1)
 @pytest.mark.abort_on_fail
-@pytest.mark.usefixtures("only_without_juju_secrets")
-async def test_admin_permissions(ops_test: OpsTest):
```

Comment:
is this test **exactly identical** to the following one?

Reply:
@Mehdi-Bendriss not sure I follow. This is only run when we do not have juju secrets. In our CIs, that is 100% of the cases

Reply:
They are 1-2-1 replicas. I will keep this comment open but close the remaining ones.